public/js/components/App.js

```diff
@@ -41,5 +46,6 @@ const App = React.createClass({
 
 module.exports = connect(
   state => ({ sources: getSources(state),
-              breakpoints: getBreakpoints(state) })
+              breakpoints: getBreakpoints(state) }),
+  dispatch => bindActionCreators(actions, dispatch)
```

Comment:
I don't see actions being used anywhere? Is this just leftover? If so, please remove it